SU-HSDO/suhumsci
# SHS-5073: Rename "Add Paragraph" to "Add Component" on Private pages

…rivate page components button

# READY FOR REVIEW

## Summary
Add component button change in Private Page content is updated from "Add Paragraph" to "Add Component"

## Steps to Test
1. Login as admin/editor
2. Add or edit Private Page content
3. Then in the Components field, the button should say "Add Component"

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
